src/exchange/account/delegation/account-delegation.html

```diff
@@ -23,15 +23,14 @@
                 <div class="form-group">
                     <label class="sr-only" for="delegationSearch"
                            data-translate="common_search"></label>
-                    <div class="oui-select oui-select_inline">
-                        <select class="oui-select__input" id="available-domains" name="available-domains"
-                                data-ng-model="ctrl.selectedDomain"
-                                data-ng-change="ctrl.onDomainValueChange()"
-                                data-ng-options="domain.displayName for domain in ctrl.availableDomains track by domain.displayName">
-                                <option value="" data-translate="exchange_all_domains"></option>
-                        </select>
-                        <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
-                    </div>
+                    <oui-select class="select-domain oui-select_inline"
```

Comment:
Maybe you can change `oui-select` to have the option to have it being inline :)

Reply:
There is no input option to configure this as of now. I have asked Axel for confirmation on this

Reply:
Yes I meant changing the source for of `oui-select` :)

Reply:
Same response from Axel. Using oui-select_inline class is the right way as of now.